app/breast-screening-reporting/2025/12/when-logins-become-dead-ends/index.md

```diff
@@ -0,0 +1,93 @@
+description:
+date: 2025-12-19
+---
+
+Lost or difficult logins stop staff being able to use the data they need. How might we prevent this?
+
+## The reality of viewing data
+
+Breast screening data is rightly kept secure behind logins. In theory, lots of data is already available, mostly in the form of raw numbers. But during our research we heard again and again from staff who were unable to access various systems and platforms due to login issues.
+
+They told us that:
+
+- logging in to one data platform had been so challenging they had now given up
+- infrequently used logins are easily lost, and some of the data they need is only published annually, six monthly or quarterly
+- they search through emails trying to find access details
+- they ask colleagues to print out data they should be able to access for themselves
+- authorisation expires between logins, requiring the process to be repeated
+- busy environments, juggling multiple tasks, with multiple systems open and lots of interruptions means access needs to be simple
+- they had applied for access to a data product or platform and never heard back
+
+We also heard from long standing breast screening programme leaders:
+
+> If they have to log in, we’ve lost them
+> -- Breast screening data expert
+
+> Even those with access enabled have trouble finding the URL. When they do find it the interface appears buggy and tells them they don't have access. They had to learn what to ignore on the way in
+> -- Former screening director
+
+> BSIS is great at giving information back to users but it’s too hard to access so most haven't explored it.
+> -- Former screening director
+
+## Learning from the past
+
+Part of our work in creating a new generation of data tools has to be to ensure that the people who need data can access it with as little friction as possible.
+
+Work has been done to improve login issues across the NHS, with smart cards and other methods, but what might this mean for our data users?
+
+We are currently looking at how to replace Breast Screening Information System (BSIS) reports with new products in the Federated Data Platform (FDP) so we looked at access to this platform for various users.
+
+## Making access easier
+
+There are two main categories when it comes to accessing the platform:
+
+1. Users with nhs.net email addresses
+2. Users with nhs.uk or other email addresses
+
+Mapping the four main FDP access journeys was enough to show us that we needed to find ways to reduce the steps users take.
+
+The longest journey looked like this:
+
+![Screenshot of a very complex flow diagram](FDP-login-non-nhsnet-no-okta-or-auth.jpg)
+
+It would only be this complex for a small minority of users, but still that was too many for our liking.
+
+## Reducing steps for our users
+
+By working with the helpful NHS FDP team at Arden and GEM who will set up our users in the background, we were able to cut the following stages that would have affected all our future users.
```

Comment:
What's GEM?

Reply:
I'm not entirely sure, and https://www.ardengemcsu.nhs.uk doesn't make it super clear either!